src/EditorFeatures/Core/CommandHandlers/InteractivePasteCommandHandler.cs

```diff
@@ -0,0 +1,105 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+extern alias InteractiveWindow;
+
+using System;
+using System.ComponentModel.Composition;
+using System.Runtime.CompilerServices;
+using System.Windows;
+using Microsoft.VisualStudio.Text.Editor;
+using Microsoft.VisualStudio.Text.Operations;
+using Microsoft.VisualStudio.Utilities;
+using Microsoft.CodeAnalysis.Editor.Commands;
+
+namespace Microsoft.CodeAnalysis.Editor.CommandHandlers
+{
+    [ExportCommandHandler(PredefinedCommandHandlerNames.InteractivePaste, ContentTypeNames.RoslynContentType)]
+    [Order(After = PredefinedCommandHandlerNames.Rename)]
```

Comment:
Can you comment these? Often, ordering of exports is about *preferences*, but in this case it's about correctness or even reliability, so I'd want people to know what they're getting into if they think about changing these.